contracts/interfaces/IGenericSwap.sol

```diff
@@ -5,7 +5,6 @@ import { GenericSwapData } from "../libraries/GenericSwapData.sol";
 
 interface IGenericSwap {
     error AlreadyFilled();
-    error InvalidTaker();
```

Comment:
We are not checking this anymore?

Reply:
Since GS order doesn't have maker sig, checking taker is not making too much sense here actually.